projects/tn-ra-manual.halex.alignment.xml

```diff
@@ -498,9 +498,20 @@ values = values.collect { value -&gt;
 	//	result = value
 	//}
 	//TODO also support parsing based on provided pattern?
+	
+	if (result == null) {
+		//try block to make sure the function is not cancelled in case an error occurs
+		try {
+			//specific format reported by RLP
+			result = Date.parse("yyyy-MM-dd'T'HH:mm:ss", value)
+		}
```

Comment:
You would need to catch an eventual exception and ignore it, otherwise it will still disrupt the execution of the code.
e.g.
```
try {
  ...
} catch (e) {
  // ignore, try next variant
}
```

Reply:
Ok, I thought in groovy, it would be possible to only use try blocks. I added them.

Reply:
@JohannaOtt Did you check if that works with the example from the error? At first glance it seems the time zone information is missing in the pattern

Reply:
![grafik](https://user-images.githubusercontent.com/34163838/90103890-90328a00-dd43-11ea-92c4-003cb4252370.png)
I checked in the groovy playground and that looked good to me.